authentication/authentication-internal-service/src/main/java/io/mosip/authentication/internal/service/integration/RestHelperImpl.java

```diff
@@ -222,26 +213,29 @@ private RestServiceException handleStatusError(WebClientResponseException e, Cla
 							"request failed with status code :" + e.getRawStatusCode(),
 							"\n\n" + e.getResponseBodyAsString());
 					List<ServiceError> errorList = ExceptionUtils.getServiceErrorList(e.getResponseBodyAsString());
-					mosipLogger.error(IdAuthCommonConstants.SESSION_ID, CLASS_REST_HELPER, "Throwing AuthenticationException",
-							errorList.toString());
+					mosipLogger.error(IdAuthCommonConstants.SESSION_ID, CLASS_REST_HELPER,
+							"Throwing AuthenticationException", errorList.toString());
 					throw new RestServiceException(errorList.get(0).getErrorCode(), errorList.get(0).getMessage(), e);
-					
+
 				} else {
-				mosipLogger.error(IdAuthCommonConstants.SESSION_ID, CLASS_REST_HELPER, METHOD_HANDLE_STATUS_ERROR,
-						"Status error - returning RestServiceException - CLIENT_ERROR -- "
-								+ e.getResponseBodyAsString());
-				return new RestServiceException(IdAuthenticationErrorConstants.CLIENT_ERROR, e.getResponseBodyAsString(),
-						mapper.readValue(e.getResponseBodyAsString().getBytes(), responseType));
+					mosipLogger.error(IdAuthCommonConstants.SESSION_ID, CLASS_REST_HELPER, METHOD_HANDLE_STATUS_ERROR,
+							"Status error - returning RestServiceException - CLIENT_ERROR -- "
+									+ e.getResponseBodyAsString());
+					return new RestServiceException(IdAuthenticationErrorConstants.CLIENT_ERROR,
+							e.getResponseBodyAsString(),
+							mapper.readValue(e.getResponseBodyAsString().getBytes(), responseType));
 				}
 			} else {
 				mosipLogger.error(IdAuthCommonConstants.SESSION_ID, CLASS_REST_HELPER, METHOD_HANDLE_STATUS_ERROR,
 						"Status error - returning RestServiceException - SERVER_ERROR -- "
 								+ e.getResponseBodyAsString());
-				return new RestServiceException(IdAuthenticationErrorConstants.SERVER_ERROR, e.getResponseBodyAsString(),
+				return new RestServiceException(IdAuthenticationErrorConstants.SERVER_ERROR,
+						e.getResponseBodyAsString(),

```

Comment:
getResponseBodyAsString() could have sensitive data ?
If there is no sensitive data in it , then you can add it as Debug log but not in the error log line.